public/index.html

```diff
@@ -0,0 +1,493 @@
+              <p>Eliefend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+            <div class="animated-dots">
+              <div class="single-dot"></div>
+              <div class="single-dot"></div>
+              <div class="single-dot"></div>
+            </div>
+          </li>
+          <li class="column-item">
+            <span class="icon-mug sm-circular-eclipse-border"></span>
+            <div class="info">
+              <h3>AND MANY MORE</h3>
+              <p>Eliefend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+            <div class="animated-dots">
+            </div>
+          </li>
+        </ul>
+      </div>
+    <div class="app-review-wrapper fifth-skew">
+      <div class="arrow-position-wrapper">
+        <span class="sm-semicircular-eclipse-border"><span class="icon-quotes"></span></span>
+        <h2 id="reviews-nav" class="center-align">App Reviews</h2>
+        <h4>Read What Our Happy Customers Are Saying</h4>
+        <span class="drawn-arrow-wh">&nbsp;</span>
+      </div>
+      <div class="review-gallery sixth-skew">
+        <div>
+          <p>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes turpis donec tincidunt tellus.</p>
+          <figure class="profile">
+            <span class="circle">
+              <img src="images/edison.jpg" alt="Portrait of Thomas Edison">
+            </span>
+            <figcaption><span class="name">Thomas Edison</span>Inventor : The Lightbulb</figcaption>
+          </figure>
+        </div>
+        <div>
+          <p>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes turpis donec tincidunt tellus.</p>
+          <figure class="profile">
+            <span class="circle">
+              <img src="images/buchanan.jpg" alt="Portrait of James Buchanan">
+            </span>
+            <figcaption><span class="name">James Buchanan</span>Winner : Best Hair</figcaption>
+          </figure>
+        </div>
+        <div>
+          <p>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes turpis donec tincidunt tellus.</p>
+          <figure class="profile">
+            <span class="circle">
+              <img src="images/stanton.jpg" alt="Portrait of Elizabeth Cady Stanton">
+            </span>
+            <figcaption><span class="name">Elizabeth Stanton</span>Champion : Women's Suffrage</figcaption>
+          </figure>
+        </div>
+      </div>
+    </section>
```

Comment:
Please review your containers ... the tabbing has me off, but I feel that things are incorrectly being set here.